js/hview.js

```diff
@@ -557,20 +584,17 @@ $(function () {
         }
     }
 
-    const renderList = function (root, vListContent = $("#vlist_content").empty(), boxContent = $("#border-box").empty()) {
+    const renderList = function (root, vListContent = document.findElementById("#vlist_content"), boxContent = document.findElementById("#border-box")) {
         $("#hview").removeClass("hide").removeClass("hidden");
         $("#main-progress").hide();
-
-        currentRootNode = root;
+        vListContent.innerHtml = ""
```

Comment:
use replaceChildren()
https://stackoverflow.com/questions/3955229/remove-all-child-elements-of-a-dom-node-in-javascript
https://developer.mozilla.org/en-US/docs/Web/API/Element/replaceChildren

Since we only care about google-chrome for now (web-usb is not supported at other places anyway)

Reply:
Done.